app/src/main/java/de/tum/in/tumcampusapp/component/ui/ticket/payload/TicketReservationCancelation.kt

```diff
@@ -0,0 +1,11 @@
+package de.tum.`in`.tumcampusapp.component.ui.ticket.payload
+
+import android.arch.persistence.room.Entity
+import android.arch.persistence.room.PrimaryKey
+import com.google.gson.annotations.SerializedName
+
+
+@Entity
+data class TicketReservationCancelation(@PrimaryKey
+                        @SerializedName("ticket_history")
+                      var ticket_history: Int = 0)
```

Comment:
Could we please use camelCase in Kotlin/Java?

Reply:
Yes I edited all the Kotlin Model classes.